README.md

```diff
@@ -155,10 +155,10 @@ non-bootable.
 
```

Comment:
@kawaii-ghost can you add a collapsable warning section indicating what these 2 options do.